ui-v2/tests/integration/services/services-test.js

```diff
@@ -0,0 +1,75 @@
+import { moduleFor, test } from 'ember-qunit';
+import { skip } from 'qunit';
+import repo from 'consul-ui/tests/helpers/repo';
+moduleFor('service:services', 'Integration | Service | services', {
+  // Specify the other units that are required for this test.
+  needs: [
+    'service:store',
+    'model:service',
+    'adapter:service',
+    'serializer:service',
+    'service:settings',
+  ],
+});
+const dc = 'dc-1';
+const id = 'token-name';
+test('findByDatacenter returns the correct data for list endpoint', function(assert) {
+  return repo(
+    'Service',
+    'findAllByDatacenter',
+    this.subject(),
+    function(stub) {
+      return stub(`/v1/internal/ui/services?dc=${dc}`, {
+        CONSUL_SERVICE_COUNT: '100',
+      });
+    },
+    function(service) {
+      return service.findAllByDatacenter(dc);
+    },
+    function(actual, expected) {
+      assert.deepEqual(
+        actual,
+        expected(function(payload) {
+          return payload.map(item =>
+            Object.assign({}, item, {
+              Datacenter: dc,
+              uid: `["${dc}","${item.Name}"]`,
+            })
+          );
+        })
+      );
+    }
+  );
+});
+skip('findBySlug returns the correct data for item endpoint', function(assert) {
```

Comment:
Ah I need to un skip this before I go about changing it - it was troublesome for some reason, will probably sort that tomorrow now, but would be good to get some eyes on this before I start changing things around.